README.md

```diff
@@ -62,9 +63,7 @@ EOF
 Test your newly created NetworkPolicy:
 
 ```bash
-$ illuminatio clean run
-```
-```
+illuminatio clean run
```

Comment:
Should this be `illuminatio run clean`? It was changed in a direct commit on master from that, can you fix it?

Reply:
The change was the other way round: https://github.com/inovex/illuminatio/commit/289a8e0a9ee351944793ee7d038c743507597e38 e.g. the command means `clean up all old illuminatio resources and then run the tests` IMHO the order is correct.